### PR TITLE
Package nocoiner.1.0.0

### PR DESCRIPTION
**Pull request by marcoonroad**
### `nocoiner.1.0.0`
A Commitment Scheme library for Coin Flipping/Tossing algorithms and sort
This project implements Commitment Schemes using the
Encrypt-then-MAC approach of authenticated encryption. Because this kind of
encryption algorithm provides both Message Confidentiality and Integrity, it fits
perfectly the Hiding and Binding properties of Commitment Schemes.
Confidentiality protects the message against passive attacks while integrity
protects it from active attacks.

---
* Homepage: https://github.com/marcoonroad/nocoiner
* Source repo: git+https://github.com/marcoonroad/nocoiner.git
* Bug tracker: https://github.com/marcoonroad/nocoiner/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by marcoonroad** [comment]
Oh, I forgot again the URL tarball and checksum hahaha. :)

**Comment by marcoonroad** [comment]
The build on my Travis-CI account is failing on format steps due mismatches of the ocamlformat version `0.8` and `0.10` among OCaml versions (from v4.03 up to v4.07).

I will update the tag `1.0.0` and checksum hash once I have fixed that, so it would be ready to merge. Thanks in advance. o/

File: packages/nocoiner/nocoiner.1.0.0/opam [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+
+run-test: ["dune" "runtest" "-p" name "-j" jobs]
+
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "dune" {build & >= "1.9"}
```
Comment (kit-ty-kate) [inline packages/nocoiner/nocoiner.1.0.0/opam]:
```suggestion
  "dune" {>= "1.9"}
```
Dune should not be flagged as a build dependency anymore (see #14266)

**Comment by kit-ty-kate** [comment]
The problem on OCaml 4.03 does not seem related. It's good to merge once the comment above is fixed.

**Comment by kit-ty-kate** [comment]
thanks a lot!

**Pull request merged**